core/types/market.go

```diff
@@ -413,6 +420,7 @@ type iProto interface {
 	iIntoProto() interface{}
 	getAsset() (string, error)
 	String() string
+	GetQuoteName() (string, error)
```

Comment:
do you mind calling it GetName rather than quoteName? quoteName sounds a bit confusing for spot.

Reply:
I hear you, but I hate it :) having getQuoteName on the product interface seems confusing to me, and I could be wrong but I don't think it's a standard name. In spot product I just call it Name. If there's anything better, happy to change to it.

Reply:
It's been a while since the future type was created, and quote name is used throughout. I do understand why it's jarring, considering a future doesn't have a quote name (at best it quotes a price, but that's besides the point). If memory serves, the `QuoteName` field here refers to the settlement asset (although I might be wrong, too). I think the idea was that a future like this would look something like this:

```
{
    SettlementAsset: "0xabcdef123456789", // asset ID
    QuoteName: "USDT", // human readable asset name
}
```

In that sense a future quote would could be easily read as 123 USDT rather than 123 0xabcdef12346789. Unless someone can point to what I'm missing here and correct me on what I think `QuoteName` refers to in this context, I'll change it to `GetAssetName()` or `GetSettlementAssetName()` or something

Reply:
GetSettlementAssetName is clearly worse from spots point of view. The documentation says:

```
"A Future product"
type Future {
  "The name of the asset (string)"
  settlementAsset: Asset!

  "String representing the quote (e.g. BTCUSD -> USD is quote)"
  quoteName: String!


```
so maybe lets stick with your current naming.

Reply:
ah, didn't check GQL for docs